python/taichi/ui/scene.py

```diff
@@ -84,6 +434,10 @@ class Scene:
     def __init__(self):
         check_ggui_availability()
         self.scene = _ti_core.PyScene()
+        warnings.warn(
```

Comment:
Please add a test in `test_deprecation.py" so that we can keep track of the removal of deprecated apis ;)

Reply:
will add in next PR that adds tests for sceneV2